GeneratorInterface/Pythia8Interface/interface/Py8InterfaceBase.h

```diff
@@ -51,6 +53,13 @@ namespace gen {
 	 unsigned int                   maxEventsToPrint;
          HepMC::IO_AsciiParticles*      ascii_io;
 
+         // EvtGen plugin
+         //
+         bool useEvtGen;
+         EvtGenDecays* evtgenDecays;
+         string evtgenDecFile;
```

Comment:
@mkirsano 
 place add the std:: namespace to the string declarations, as eventually the person whose header you include  will fix their code and remove the using std from their header..